addons/website/static/src/js/editor/widget_link.js

```diff
@@ -47,6 +50,10 @@ weWidgets.LinkTools.include({
      */
     _adaptPageAnchor: function () {
         const urlInputValue = this.$('input[name="url"]').val();
+        if (EMAIL_REGEX.test(urlInputValue)) {
+            // Don't try to load anchors for email addresses.
+            return;
```

Comment:
as @rdeodoo told me, maybe we need that for `tel:xxx` links too but as I can't add such a link in 16.0, difficult to be sure it's needed :/